insilico_panels/KK/lipid.v1.0/lipid.v1.0

```diff
@@ -0,0 +1,6 @@
+LPL
+APOC2
+APOA5
+LMF1
+GPIHBP1
+
```

Comment:
Should be version 2.0. See comment below

Reply:
Does that mean that the directory name should also change to v2.0? (Like KK/lipid.v2.0/lipid.v2.0?)

Reply:
Should it? This panel for example contains a part of the genes from "mixed small panels" but it's still a new panel right?
Then there are other panels, like neurometabola sjukdomar where I have just added genes to the existing list.

Reply:
I took this example because Anders made the comment here - my point was about whether directory and file names should have the same version (I think so). But you are right, perhaps this only applies to the two panels that changes name and/or got genes added.

Reply:
Sorry, I agree that the directory and files should have the same name. But I'm not sure whether the "new" panels should be v2.

Reply:
I think they should be 1.0 since they are new. It is a bit awkward having the version name in the folder name, but I think it's too annoying to change this at this point.